### PR TITLE
[FRONT-109] Add custom 404 page

### PR DESCRIPTION
**Pull request by mohammad-akkad**
### Description
Added customized 404 page

### How I implemented
*I had to mention the designer and the website because it is a condition for free download

image

### CODE REVIEW

**Review by kigadye — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -33,7 +33,6 @@ module.exports = {
         purple: '#BC88FD',
         blue: '#4FA8FD',
         green: '#4DB783',
-
```
Comment (ArnoClaude) [inline tailwind.config.js:36]: avoid (formatting) changes in unrelated files

**Review by ArnoClaude — APPROVED** [review]
